src/pow.cpp

```diff
@@ -355,12 +355,16 @@ void multinomial_coefficients_mpz(int m, int n, map_vec_mpz &r)
 RCP<const Basic> pow_expand(const RCP<const Pow> &self)
 {
     RCP<const Basic> _base = expand(self->base_);
+    bool negative_pow = false;
     if (! is_a<Integer>(*self->exp_) || ! is_a<Add>(*_base))
         return self;
```

Comment:
Although `_base` is a `Add`, that does not guarantee that `result` is not a `Mul` or a `Number` ...
`(1/(sqrt(3)*sqrt(2) + sqrt(6)*I)**4).expand()`